src/mono/mono/eventpipe/ep.c

```diff
@@ -1,6 +1,6 @@
 #include <config.h>
 
-#ifdef ENABLE_PERFTRACING
+#if defined(ENABLE_PERFTRACING) && !defined(DISABLE_EVENTPIPE)
```

Comment:
if ENABLE_PERFTRACING is not defined when DISABLE_EVENTPIPE is, it is enough and inline with rest of code to only check that in eventpipe code (since it might potentially be shared with coreclr going forward).